tests/helper/helper_kubectl.go

```diff
@@ -138,9 +139,12 @@ func (kubectl KubectlRunner) GetServices(namespace string) string {
 	return output
 }
 
-// CreateRandNamespaceProject create new project with random name in kubernetes cluster (10 letters)
+// CreateRandNamespaceProject create new project with test file name, line number and 10 random letters
 func (kubectl KubectlRunner) CreateRandNamespaceProject() string {
-	projectName := RandString(10)
+	//Get current test filename and remove file path, file extension and replace undescores with hyphens  
+	currGinkgoTestFileName := strings.Replace(CurrentGinkgoTestDescription().FileName[strings.LastIndex(CurrentGinkgoTestDescription().FileName, "/")+1:strings.LastIndex(CurrentGinkgoTestDescription().FileName, ".")], "_", "-", -1)
+	currGinkgoTestLineNum := strconv.Itoa(CurrentGinkgoTestDescription().LineNumber)
+	projectName := currGinkgoTestFileName + "-lnum-" + currGinkgoTestLineNum + "-" + RandString(10)
```

Comment:
These lines are the same for both oc and k8s, perhaps we could consider a helper function that returns the projectName.

Reply:
agreed, moved to helper_generic.go